go/vt/vtgate/tabletgateway_test.go

```diff
@@ -84,7 +85,7 @@ func TestTabletGatewayBeginExecute(t *testing.T) {
 
 func TestTabletGatewayShuffleTablets(t *testing.T) {
 	hc := discovery.NewFakeHealthCheck(nil)
-	tg := NewTabletGateway(context.Background(), hc, nil, "local")
+	tg := NewTabletGateway(context.Background(), hc, nil, "cell1")
```

Comment:
I don't really understand how the existing `shuffleTablets` test cases where even passing? 🤔